integration-tests/src/test/groovy/com/okta/sdk/tests/it/UsersIT.groovy

```diff
@@ -620,6 +627,7 @@ class UsersIT extends ITSupport {
 
         assertThat(updatedUser.lastUpdated, greaterThan(originalLastUpdated))
         assertThat(updatedUser.getProfile().getProperties().get("nickName"), equalTo("Batman"))
+        //assertThat(updatedUser.getProfile().getAdditionalProperties().get("key1"), equalTo("val1"))
```

Comment:
Commented this and above because the ITs will be run in several Org cells as part of PDV. For this test to run successfully, the profile property must be added to the User schema first. Since this is not a guarantee, we are disabling this test to prevent IT failures under such circumstances.